lib/matplotlib/pyplot.py

```diff
@@ -557,6 +564,9 @@ def make_active(event):
         if _INSTALL_FIG_OBSERVER:
             fig.stale_callback = _auto_draw_if_interactive
 
+    if clear:
+        figManager.canvas.figure.clear()
```

Comment:
:bike: :house:  `fig` is a ref to the current figure that we already have just above.